lang/zh-tw.yml

```
@@ -1,197 +1,198 @@
-# CoreProtect Language File (zh-TW)
+# CoreProtect 語言檔案（zh-TW）
+# Translator: whitebear13579, Flandre_tw
 
 ACTION_NOT_SUPPORTED: "此指令不支援該操作。"
-AMOUNT_BLOCK: "{0} {方塊|方塊}"
-AMOUNT_CHUNK: "{0} {區塊|區塊}"
-AMOUNT_ENTITY: "{0} {實體|實體}"
-AMOUNT_ITEM: "{0} {物品|物品}"
-API_TEST: "API 測試成功!"
-CACHE_ERROR: "警告!: 驗證快取 {0} 時出現錯誤。"
-CACHE_RELOAD: "強制從資料庫中載入 {鏡像|世界} 快取。"
-CHECK_CONFIG: "請檢查 config.yml"
-COMMAND_NOT_FOUND: "未知的指令。"
-COMMAND_THROTTLED: "你已受到速率限制，請稍後重試。"
-CONTAINER_HEADER: "儲存容器已轉換。"
-DATABASE_BUSY: "資料庫請求過多，請稍後在重試。"
-DATABASE_INDEX_ERROR: "無法驗證的資料庫索引條件。"
-DATABASE_LOCKED_1: "資料庫已被鎖定，請等待15秒後在重試。"
-DATABASE_LOCKED_2: "資料庫正在使用中，請稍後在試一次。"
-DATABASE_LOCKED_3: "若需要將資料庫鎖定的功能關閉，請設定 \"database-lock: false\"."
-DATABASE_LOCKED_4: "關閉資料庫鎖定功能可能導致資料造成不可逆的損壞!"
-DATABASE_UNREACHABLE: "資料庫無法存取，請刪除資料並關閉。"
-DEVELOPMENT_BRANCH: "你使用的版本為開發的分支版本，已跳過補丁腳本。"
-DIRT_BLOCK: "已在你的身下放置一個泥土塊。"
-DISABLE_SUCCESS: "執行成功!已關閉 {0}"
-ENABLE_FAILED: "{0} 無法成功啟動。"
-ENABLE_SUCCESS: "{0} 已經成功開啟。"
-ENJOY_COREPROTECT: "喜歡 {0}嗎? 加入我們的開發Discord!"
-FINISHING_CONVERSION: "完成資料轉換中，請稍後..."
-FINISHING_LOGGING: "完成數據紀錄中，請稍後..."
-FIRST_VERSION: "初始化資料庫: {0}"
-GLOBAL_LOOKUP: "請不要指定一個範圍來進行整個世界的搜尋。"
-GLOBAL_ROLLBACK: "使用 \"{0}\" 來進行全面的 {回朔|恢復}"
-HELP_ACTION_1: "將某個尋找的東西限制。"
-HELP_ACTION_2: "範例: [a:block], [a:+block], [a:-block] [a:click], [a:container], [a:inventory], [a:item], [a:kill], [a:chat], [a:command], [a:sign], [a:session], [a:username]"
-HELP_COMMAND: "顯示該指令的更多訊息。"
-HELP_EXCLUDE_1: "排除 方塊 / 玩家"
-HELP_EXCLUDE_2: "範例: [e:stone], [e:Notch], [e:stone,Notch]"
-HELP_HEADER: "{0} 幫助"
-HELP_INCLUDE_1: "包括指定的 方塊 / 實體"
-HELP_INCLUDE_2: "範例: [i:stone], [i:zombie], [i:stone,wood,bedrock]"
-HELP_INSPECT_1: "當檢查器啟用後，你可以執行以下操作:"
-HELP_INSPECT_2: "左鍵點擊你欲查詢的方塊，可以查看誰曾經破壞過該方塊。"
-HELP_INSPECT_3: "右鍵點擊你欲查詢的方塊，可以查看相鄰的方塊曾被破壞過的訊息。"
-HELP_INSPECT_4: "在欲查詢的位置放置一個方塊，可以查看該位置曾被破壞過哪個方塊。"
-HELP_INSPECT_5: "在液體(水、岩漿...)中放置一個方塊，可以查看是誰放置此液體的。"
-HELP_INSPECT_6: "右鍵點擊門、箱子等物品，可以查看玩家的使用紀錄。"
-HELP_INSPECT_7: "提示:你可以使用 \"/co i\" 來加快尋找的速度。"
+AMOUNT_BLOCK: "{0} 個{方塊|方塊}"
+AMOUNT_CHUNK: "{0} 個{區塊|區塊}"
+AMOUNT_ENTITY: "{0} 個{實體|實體}"
+AMOUNT_ITEM: "{0} 個{物品|物品}"
+API_TEST: "API 測試成功。"
+CACHE_ERROR: "警告 ： 驗證快取 {0} 時出現錯誤。"
+CACHE_RELOAD: "正在強制從資料庫中重新載入{鏡像|世界}快取。"
+CHECK_CONFIG: "請檢查 config.yml 檔案。"
+COMMAND_NOT_FOUND: "未知的指令 \"{0}\"。"
+COMMAND_THROTTLED: "請稍後再試。"
+CONTAINER_HEADER: "儲存容器變更記錄。"
+DATABASE_BUSY: "資料庫繁忙，請稍後再試。"
+DATABASE_INDEX_ERROR: "無法驗證資料庫索引。"
+DATABASE_LOCKED_1: "資料庫已被鎖定，請等待 15 秒後再試……"
+DATABASE_LOCKED_2: "資料庫正在使用，請稍後再試一次。"
+DATABASE_LOCKED_3: "若要關閉資料庫鎖定功能，請設定 \"database-lock: false\"。"
+DATABASE_LOCKED_4: "關閉資料庫鎖定功能，可能導致資料損壞。"
+DATABASE_UNREACHABLE: "資料庫無法存取，正在拋棄資料並關閉。"
+DEVELOPMENT_BRANCH: "檢測到正在使用開發分支的版本，已跳過使用補丁腳本。"
+DIRT_BLOCK: "已在你的下方放置了泥土。"
+DISABLE_SUCCESS: "執行成功 ！ 關閉 {0} ！"
+ENABLE_FAILED: "無法啟用 {0}。"
+ENABLE_SUCCESS: "已經成功啟用 {0} ！"
+ENJOY_COREPROTECT: "喜歡 {0} 嗎 ？ 加入我們的 Discord ！"
+FINISHING_CONVERSION: "正在完成資料轉換，請稍後……"
```

```suggestion
FINISHING_CONVERSION: "正在完成資料轉換，請稍候……"
```
`稍後` means `a moment later`. e.g. `稍後再試`
`稍候` means `wait a moment`. e.g. `請稍候`